config/default.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
       revisions: 'revisions',
       save: 'save',
     },
+    plugins: {
```

Comment:
agreed. plugabilly expects `config.search` to be an array

So that means in content types we must provide an array. And it should check if what it sends to plugabilly is an array first